extensions/sql-migration/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "sql-migration",
   "displayName": "%displayName%",
   "description": "%description%",
-  "version": "1.4.1",
+  "version": "1.4.2",
```

Comment:
I preemptively vbumped to 1.4.1 as part of my last PR, but that hasn't been released yet. I don't think you need to bump it again right now